### DISCUSSION TITLE
Git_blacklist add exclude ignore_patterns

### DISCUSSION THREAD

**Discussion by Mte90** [phpro/grumphp#901] [Ideas]
| Q               | A
| --------------- | ---
| Version         | `vendor/bin/grumphp -V`
| Bug?            | no
| New feature?    | yes
| Question?       | no
| Documentation?  | no
| Related tickets | comma-separated list of related tickets

The git_blacklist task doesn't have ignore_patterns like many others tasks.
https://github.com/phpro/grumphp/blob/master/doc/tasks/git_blacklist.md

In my case I don't want to put the path for all the files/folders but I just want to exclude one single file.

**Comment by ghost** [comment]
Not tested yet: https://github.com/phpro/grumphp/pull/1005 but it looked pretty straight forward.
